template.go

```diff
@@ -52,16 +54,29 @@ func NewTemplateParser(cfg Config) *TemplateParser {
 			}
 			t.Vars[strings.TrimSuffix(filepath.Base(settingsFile.Name()), ".json")] = v
 		}
+	} else if cfg.Templates != "" {
```

Comment:
why? i need using cfg.Settings & cfg.Templates together

Reply:
ouch, you're right

Reply:
please, check #10